src/app/zap-templates/zcl/data-model/chip/general-commissioning-cluster.xml

```diff
@@ -115,48 +123,66 @@ limitations under the License.
         </mandatoryConform>
       </otherwiseConform>
     </attribute>
-    <command source="client" code="0x00" name="ArmFailSafe" response="ArmFailSafeResponse" optional="false" cli="chip fabric_commissioning armfailsafe">
-      <description>Arm the persistent fail-safe timer with an expiry time of now + ExpiryLengthSeconds using device clock</description>
-      <arg name="ExpiryLengthSeconds" type="int16u"/>
-      <arg name="Breadcrumb" type="int64u"/>
+    <attribute code="0x000A" side="server" name="RecoveryIdentifier" define="RECOVERY_IDENTIFIER" type="octet_string" length="8" minLength="8" default="0" optional="true">
```

Comment:
Shouldn't this have `apiMaturity` bits too?

Reply:
Yeah, I was using an older version of Alchemy that handled that in a different way. Just uploaded the new version.